src/map/MapVEuMap.tsx

```diff
@@ -186,6 +188,7 @@ function MapVEuMap(props: MapVEuMapProps, ref: Ref<PlotRef>) {
     showSpinner,
     showNoDataOverlay,
     showScale = true,
+    scrollingEnabled,
```

Comment:
Hi @moontrip - please could you make this default to true, then I would like to approve and merge this PR and use the new prop for the FSM minimap.
I might do this before you log on for the day. Let's keep in touch via Slack.
```suggestion
    scrollingEnabled = true,
```

Reply:
Actually, I think I was trying to kill two birds with one stone.

Disabling scroll-wheel functionality isn't enough for the minimap.

I'll add an `interactive` (defaults to true) prop for MapVEuMap in my fsm-minimap branch.

Still happy for this scrollingEnabled prop to be added though :-)

Reply:
@bobular this took some time to resolve merge conflicts, but I have applied your suggestion to the lasted commit 👍